deployment/helm/base/templates/developer-storage.yaml

```diff
@@ -0,0 +1,84 @@
+# This is a sample PersistentVolume that is expected to be present.  It can be anything, so long as it can be bound by a PersistentVolumeClaim
+# when Skaha is installed.  This is done by matching th labels.  The intent is that a Cluster Administrator will have a PersistentVolume setup,
+# and will direct deployers on what label to use to bind to it.
+# This will be used by the User Sessions when starting up to gain access to the shared Storage with the Skaha service.
+# This default will create a hostPath volume with no declared PV.
+
+{{- if .Values.developer.storage.path}}
```

Comment:
This assumes that if the `developer` stanza exists, then the `storage.path` will also.  One could actually just declare `developer:` and leave it at that and it would be valid.
To error proof this, you could say:
```yaml
{{- with .Values.developer }}  # If this is absent, then ignore this block
{{- $storagePath := .storage.path | required ".Values.developer.storage.path must be set to underlying storage folder" ." -}} # Already in the context of .Values.developer, so just saying .storage.path is correct.
...
  path: {{ $storagePath }}
...
{{- end }}
```

Reply:
The `developer` stanza will always exist, both in production and developer mode, since its now in the base `values.yml`.

However, `storage.path` only exists in developer mode, through `dev.overrides.yml` and this logic breaks the production deployment of helm,

```bash
helm template .
```
```bash
Error: execution error at (base/templates/developer-storage.yaml:9:38): developer.storage.path is required for developer mode
```